### ISSUE TITLE
Barrier

### ISSUE THREAD

**Issue by vermgaur** [googlecodelabs/constraint-layout#30]
Barrier does not display when we right click on the blue print.

**Comment by errjun7** [comment]
Yes, that's because the barrier feature is *not* mentioned in the menu in **Android 2.3**.
You can *still* use it in the *XML* file though.
If you want to see it in the menu as well, you would have to download the **Android Studio 3.0** beta version, as **Codelabs** had mentioned in the beginning(https://codelabs.developers.google.com/codelabs/constraint-layout/index.html?index=..%2F..%2Findex#0) of this tutorial.
You can continue using the 2.3 version alongside the 3.0 version.
